### ISSUE TITLE
LAN servers have no titles

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#25]
```
When creating a server, there is an empty "title" column created in the server 
list. Only the server's IP and port is shown.

From my limited usability testing on this, it seems users don't always visually 
see the server when there's only the local IP address on the far right and no 
server name.

As far as I can tell, there doesn't seem to be any (easy?) way to set the 
server title when creating a LAN server, so I'd suggest that Hedgewars simply 
uses the machine's hostname by default. So if my computer is called kusanagi, 
it will show up as 

"kusanagi   |  192.168.1.133  |  46631 "
```

-----
Original issue reported on code.google.com by `nekoh...@gmail.com` on 31 Aug 2010 at 4:59

Attachments:
* [Capture.png](https://storage.googleapis.com/google-code-attachments/hedgewars/issue-25/comment-0/Capture.png)

**Comment by GoogleCodeExporter** [comment]
```
If you use the frontend's button to start/create a LAN server you'll be asked 
to input the server's name - but that's somehow lost either not set or not 
submitted or whatever.
```

-----
Original comment by `mario.li...@gmail.com` on 2 Sep 2010 at 1:10
* Changed state: **Accepted**
* Added labels: **Priority-Low**
* Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Indeed, you can set the title before creating the server, but this gets lost so 
the title isn't shown.
```

-----
Original comment by `nekoh...@gmail.com` on 2 Sep 2010 at 3:52

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 7b65e83e261d.
```

-----
Original comment by `unC0Rr` on 15 Apr 2011 at 5:37
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in rev 7b65e83e261d
```

-----
Original comment by `unC0Rr` on 15 Apr 2011 at 5:37

**Comment by GoogleCodeExporter** [comment]
```
Verified (title shows up)
```

-----
Original comment by `jerryma1...@gmail.com` on 26 Nov 2011 at 7:58
* Changed state: **Verified**
